### DISCUSSION TITLE
I love the product! But that name...

### DISCUSSION THREAD

**Discussion by amandahawkpdx** [sct/overseerr#2375] [General]
Hi there, I wanted to address an elephant in the room. I felt some discomfort passing along an overseerr URL to a Black friend who agreed that the use of the word is tone-deaf at best. 

I thought your team may not be US-based and may not be aware that the word "overseer" was used to describe slave drivers in the 18th century in the US. The connotation still stands today and I just want to lean in and support a product I enjoy in achieving the welcoming of all cultures.

Thank you for reading, and thank you for making a really cool product. Happy holidays!

**Comment by TheCatLady** [comment]
@amandahawkpdx Thank you for bringing this to our attention!  I personally was not aware of the history or negative connotation of the word "overseer," and really appreciate you taking the time to raise this issue.  The name is not intended to be offensive, but we also recognize that the lack of ill intent does not mean that the name is not hurtful/racist.

The team is discussing a potential name change internally.  Please note that _if_ a name change does happen, it will definitely take some time.

On a similar note, there _are_ plans to rename our `master` branch to `main` in the nearer term.  This is something we have been meaning to do, but just haven't gotten around to yet (you might have noticed that the team has been busier and less active in recent months).

Happy holidays to you as well!

**Comment by amandahawkpdx** [comment]
Thank you @TheCatLady for your thoughtful response. I wanted to offer some suggestions that were made while discussing with friends/family how great your team is for being open to the change. We wanted to help with the potential new branding for the product :)

Privateerr - would be a nod to the origin of the platform :)

Seerr - a suggestion made by a young person to whom I described the function of the product

Tubularr - offered along with the following suggestion for a commercial, which I love:

Scene start: Tubes, wires, generally geeky stuff sparking and zooming all Tron like, then it's some kind of torrent matriarch monster that is badass and surfing said sparking computer network stuff and we call it tubular

We had fun coming up with ideas, and I am glad we could help to spark the conversation!

**Reply by sct** [reply]
Seerr is interesting because we just need to drop the Over part. 🤔

**Reply by rdvo** [reply]
yeah for sure!

**Reply by DCMarine** [reply]
It is worth to note, that it may be easier to do what Ombi does, with the "Application" name updating the title of the website and the title on the left top. Which then puts the control in the hands of the administrator of the installed OverSeerr to come up with a name for their own instance.

**Reply by jlpouffier** [reply]
I just wanted to add that a seer in the Norse mythology is someones that shapes the past, the present and the future.
(Not exactly this grammar but close)

I find this name quite fitting and badass

**Comment by DolfLund** [comment]
Some other ideas 

Seekrr
Findrr
Browsrr
Perusrr
Discoverr

(Sorry if they seem cheesy)

**Comment by CaesarBishop** [comment]
I'm sorry, I get where everyone is coming from but changing the name is a dumb idea.  As a person of color I can tell you it's not about the word its about the context.  There is no negativity intended here in my opinion so I see no need to change the name.

No one is going to stop using your product because they see the word "Overseer".

**Reply by tdeverx** [reply]
You're both correct and incorrect, while context _is_ important you unfortunately don't speak for every person of colour. You're 100% entitled to your opinion and while I agree no negativity is intended, it doesn't remove the fact that some may take it as such. I think changing the name would be a great, inclusive & future proofed idea which I totally support.

**Reply by TheCatLady** [reply]
@CesarBishop Please see my previous post in this discussion: "The name is not intended to be offensive, but we also recognize that the lack of ill intent does not mean that the name is not hurtful/racist."

A name change is planned; the team has just been busy with real life things. Hopefully soon!

**Reply by CaesarBishop** [reply]
Ok then I cast my vote for Tubularr because everytime I say it I giggle

**Comment by amandahawkpdx** [comment]
This word wears its context like an albatross...definitely not about how I "feel" about it, it is a fact that the word is tied directly to American slavery. Thanks for listening, devs.

overseer (n.)
late 14c., "supervisor, superintendent, one who looks over," agent noun from [oversee](https://www.etymonline.com/word/oversee?ref=etymonline_crossreference#etymonline_v_10089) (v.). Specifically, "one who superintends workmen;" especially with reference to slavery, "one who has charge, under the owner or manager, of the work done on a plantation."
